cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinSecondaryCategory.java

```diff
@@ -10,7 +10,7 @@
 public enum CristinSecondaryCategory {
     ANTHOLOGY("ANTOLOGI", "ANTHOLOGY"),
     MONOGRAPH("MONOGRAFI", "MONOGRAPH"),
-    JOURNAL_ARTICLE("ARTIKKEL_FAG", "JOURNAL_ARTICLE"),
+    JOURNAL_ARTICLE("ARTIKKEL_FAG", "JOURNAL_ARTICLE", "ARTIKKEL", "ARTIKKEL_POP", "OVERSIKTSART"),
```

Comment:
This is our internal mapping of the Cristin categories. I think that we should not compress the cristin categories and do in practice the mapping to the NVA catogory here. It is better to create more categories with names based loosely on the cristin english names and perform the mapping in the CristinMapper, i.e., extend the `isJournal()` method.